packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -492,20 +492,38 @@ class _CupertinoAlertDialogState extends State<CupertinoAlertDialog> {
   }
 }
 
-/// Rounded rectangle surface that looks like an iOS popup surface, e.g., alert dialog
-/// and action sheet.
+/// An iOS-style rounded rectangle surface useful for showing modal overlays,
+/// such as dialogs and action sheets.
 ///
-/// A [CupertinoPopupSurface] can be configured to paint or not paint a white
-/// color on top of its blurred area. Typical usage should paint white on top
-/// of the blur. However, the white paint can be disabled for the purpose of
-/// rendering divider gaps for a more complicated layout, e.g., [CupertinoAlertDialog].
-/// Additionally, the white paint can be disabled to render a blurred rounded
-/// rectangle without any color (similar to iOS's volume control popup).
+/// By default, this surface is composed of a transparent surface color painted
+/// atop a [BackdropFilter]. The surface color is light gray when the ambient
+/// [CupertinoTheme] brightness is [Brightness.light], and dark gray when the
+/// ambient brightness is [Brightness.dark]. The [BackdropFilter] applied
+/// beneath the surface saturates and blurs underlying content.
```

Comment:
I suggest replacing the last two paragraphs with this.
```suggestion
/// beneath the surface saturates and blurs underlying content.
/// In other words, the overall effect is result = add_surface_color(blur(saturate(source))).
```

Reply:
Done! Reads a lot better now.